src/main/resources/lib/menu.js

```diff
@@ -104,33 +104,44 @@ exports.getBreadcrumbMenu = function (params = {}) {
  * Get menu tree
  * @param {integer} levels - menu levels to get
  * @param {Object} params - configure the end result. see getSubMenus
- *   @param {String} [params.urlType=Server] - Control type of URL thats generated in getSubMenus.
-
- * @returns {Array}
+ * @param {String} [params.ariaLabel="menu"] - The aria label added to the nav element
+ * @returns {Object} 
+ * @returns {Array} object.menuItems The list of menuItems and children
+ * @returns {String} object.ariaLabel The ariaLabel used for this menu
  */
 exports.getMenuTree = function (levels, params) {
-    let menu = [];
-    let site = libs.portal.getSite();
+    if (params.ariaLabel == undefined)
+        params.ariaLabel = "menu";
+    const site = libs.portal.getSite();
+    let menuItems = [];
     if (site) {
-        menu = getSubMenus(site, levels, params);
-    }
-
-    return menu;
+        menuItems = getSubMenus(site, levels, params);
+    } 
+    
+    return {
+        menuItems,
+        ariaLabel: params.ariaLabel,
```

Comment:
`ariaLabel: params.ariaLabel || "menu"` ? Da kan muteringen `params.ariaLabel = "menu";` droppes.

Reply:
Liker ikke å sette default verdier på slutten av en funksjon. 
Tanken her er egentlig å sette en default verdi på toppen som i `function(levels=1, params={?}, ariaLabel="menu")`
Men i et object. Tingen er at jeg ikke har lyst til å ha metoden altså `function ( ...allcodehere...)`
Ser for meg at kanskje
`
function (levels, params={ ariaLabel: "menu })` kan være lov å skrive, men hva med alle de andre verdiene som kan være med? hvordan defaulter vi dem / ikke defaulter vi dem da?

Reply:
Må det ikke være en verdi der fra før hvis det er mutering?

Reply:
Hmm, kan være verdt å se nærmere på best practices her. 

Sidenote: jo, params er jo allerede et objekt og muteres dermed hvis den får lagt til et felt.